cli.js

```diff
@@ -3,14 +3,17 @@
 var meow = require('meow');
 var killTabs = require('./');
 
-meow({
+var cli = meow({
 	help: [
-		'Usage',
-		'  $ kill-tabs'
+		'Usage: kill-tabs [OPTIONS]',
+		'  --no-chromium       Do not kill tabs from chromium',
```

Comment:
@arthurvr see https://github.com/sindresorhus/kill-tabs/pull/4#issuecomment-124905709 I can add that afterwards.